test/shared/src/main/scala/zio/test/laws/ZLawsF.scala

```diff
@@ -196,6 +196,16 @@ object ZLawsF {
       final def run[R1 <: R, F[-_]: CapsF, A: Caps](genF: GenF[R1, F], gen: Gen[R1, A]): ZIO[R1, Nothing, TestResult] =
         checkM(genF(gen), genF(gen), genF(gen))(apply(_, _, _).map(_.map(_.label(label))))
     }
+
+    /**
+     * Constructs a law from a pure function taking three parameters.
+     */
+    abstract class Law3Function[-CapsF[_[-_]], -Caps[_]](label: String) extends Contravariant[CapsF, Caps, Any] {
```

Comment:
This is always a function compose law, so maybe `LawCompose` or `LawFunctionCompose`?

Reply:
The other possibility is to push it into ZIO Prelude as a class in the companion object of the associated type class (Covariant).

Reply:
I like the renaming. Pushing it to ZIO Prelude is interesting. The disadvantage there would be that it is less ergonomic if someone else wants to implement their own law with the same shape, or even if we want to do so somewhere else in ZIO Prelude. But it does solve the problem that we don't necessarily know at the ZIO Test love the shape of all the laws we will want to test.